pkg/block/blkid.go

```diff
@@ -0,0 +1,169 @@
+// Copyright (C) 2015-2020 the u-root Authors. All Rights Reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+//
+// SPDX-License-Identifier: BSD-3-Clause
+//
+
+//Package block contains functions dealing with linux block devices and the underlying hardware.
+package block
```

Comment:
we have a pkg/storage that does some of this stuff natively, without using C backup. I'd probably prefer that this pkg/block not enter u-root, and pkg/storage be extended with what we need.